packages/web/src/components/collection/desktop/RepostsFavoritesStats.tsx

```diff
@@ -0,0 +1,76 @@
+import { useCallback, MouseEvent } from 'react'
```

Comment:
Julian specifically requested this component be a one off for the moment; not replacing our existing `RepostsFavoritesStats` component for the moment. I kept the name the same and put it inline